docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
 
 ### Internet backups: Git & Github
 
 #### Git 101
 
 ?> **Git** is a tool that makes it easier to track changes to files. When you edit a file, git can help you determine exactly *what* changed, *who* changed it, and *why*. <br /><br />It is useful for coordinating work among multiple people on a project, and for tracking progress over time by saving “checkpoints”. You could use it while writing an essay, or to track changes to artwork and design files. [\[1\]](https://hackernoon.com/understanding-git-fcffd87c15a3)
 
-For our purpose, **git** will be our mean to:
+**For our purposes**, ***git*** will be our mean to:
 - Create versions of our collection (*\"checkpoints\"*).
 - Upload\* our work to a safe\*\*, remote place.
 - *(Bonus)* Synchronize our work between devices.
 
-\*: GitHub has a 1GB limit on free private repositories.
-\*\*: *Although GitHub can be considered fairly reliable, we highly recommend that you implement a solution to encrypt your collection. This is, after all, a way to your most inner thoughts.*
+\*: GitHub has a 1GB limit on free private repositories. [GitLab](https://about.gitlab.com) offers 10gb free storage by default and [BitBucket](http://bitbucket.org) offers a 5gb plan for 3$. 
+
+\*\*: *Although GitHub can be considered fairly reliable, we highly recommend that you implement a solution to encrypt your collection. This is, after all, a way to your innermost thoughts.*
 
 #### Installing & Setting up Git with GitHub
 
-1. Create a [GitHub account](https://github.com/join/)
-2. [Follow this guide](https://vladmihalcea.com/tutorials/git/windows-git-ssh-authentication-to-github/) to:
-  - Install Git
-  - Setup a SSH key that automatically authenticate your Git installation with your GitHub account
+!> If you want to use SSH authentication instead of HTTPS, follow [this guide](https://vladmihalcea.com/tutorials/git/windows-git-ssh-authentication-to-github/)
+
+1. Create a [GitHub account](https://github.com/join/).
+
+2. Download [Git for Windows](https://gitforwindows.org/).
+
+3. Install it with default settings
+
+5. Open a command prompt: type `cmd.exe`, then press <kbd>Enter</kbd> and run the following two commands (with your own name and email you use for github):
+
+   ```
+   git config --global user.name "Piotr Wozniak"
+   git config --global user.email woz@supermemo.org
+   ```
+   Setup is now complete!
   
 #### Creating & Synchronizing your GitHub repository
 
-?> **In this example, the GitHub repository name will be named SuperMemo-Collection**
+?> **In this example, the GitHub repository we create will be named SuperMemo-Collection**
 
-1. Create a [new repository](https://github.com/new)
+1. Create a [new repository](https://github.com/new):
   - Give a name to your new repo (e.g. *SuperMemo-Collection*)
   - Select **Private**
+  - It is recommended to click **Initialize this repository with a README** to be able to immediately clone to computer
```

Comment:
You should be able to clone the repository even without a README

Reply:
I will remove this in the live version as I am confident about this information. Feel free to revert my change if I am mislead.